lib/db/mysql/patches/patch-003-004.sql

```diff
@@ -0,0 +1,15 @@
+-- Adds support for Client Developers for OAuth clients
+
+CREATE TABLE IF NOT EXISTS clientDevelopers (
+  developerId INT NOT NULL,
```

Comment:
Suggest a bytestring id rather than integer id here, for consistency with `clientId` and others.